src/ringmesh/basic/logger.cpp

```diff
@@ -42,6 +42,6 @@
 
 namespace RINGMesh
 {
-    std::mutex Logger::lock_;
+//    std::mutex Logger::lock_;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this block of commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=cpp%3ACommentedCode)